### DISCUSSION TITLE
Igx-calendar not working in Angular 14 version.

### DISCUSSION THREAD

**Discussion by swapnilahirwar28** [IgniteUI/igniteui-angular#12110] [General]
## Question  
For questions, please use the `Discussions` tab of the repository!
This is my code in previous angular and it's working fine :-
<article class="calendar-wrapper">
                                            <igx-calendar
                                                (selected)="onSelection($event)"
                                                selection="multi"></igx-calendar>
</article>

But why is it not working in angular 14 version??

**Comment by ChronosSF** [comment]
@swapnilahirwar28 , we can't see your code. Could you please try to paste it or attach it again?

**Comment by hanastasov** [comment]
```html
<article class="calendar-wrapper">
                                            <igx-calendar
                                                (selected)="onSelection($event)"
                                                selection="multi"></igx-calendar>
</article>
```

This is the code

**Comment by hanastasov** [comment]
@swapnilahirwar28 , please check this sample https://stackblitz.com/edit/angular-kkbukj?file=src/app/calendar-sample-5/calendar-sample-5.component.ts - it seems to work fine.
